src/mumble/ConfigWidget.cpp

```diff
@@ -1,3 +1,4 @@
+#include "ConfigWidget.h"
```

Comment:
This include should go after the license header (see how it is done in other files)

Reply:
My bad. This was not expected. It came by mistake. Correcting it.